core/src/util/time.rs

```diff
@@ -1,6 +1,40 @@
-use std::time::{SystemTime, UNIX_EPOCH};
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
 
-/// Calculate a timestamp in seconds, rounded to the nearest 1000
+const DAYS: u64 = 60 * 60 * 24;
```

Comment:
Maybe `SECS_PER_DAY` would be a better name?
Or we could use this: https://docs.rs/arrow2/latest/arrow2/temporal_conversions/constant.SECONDS_IN_DAY.html
(though it seems ridiculous to add a dependency just for time constants)

Reply:
Will rename it in my future PR.